.github/workflows/close-stale.yaml

```diff
@@ -17,6 +17,34 @@
         default: '.'
         required: false
         type: string
+      days_before_issue_stale:
```

Comment:
I think you missed adding these inputs to the actual job instead of the hard-coded values that are there now...